### ISSUE TITLE
Primary & foreign key referencing same column panics

### ISSUE THREAD

**Issue by callym** [SeaQL/sea-orm#186]
Hi, I've got the following SQL, which crashes the generator. 
Commenting out the foreign key line, or changing the primary key to `PRIMARY KEY (a)` makes it work fine, but I'm not sure why, as I think what I'm doing is valid?

```sql
CREATE TABLE IF NOT EXISTS foo (
  a TEXT NOT NULL,
  PRIMARY KEY (a)
);

CREATE TABLE IF NOT EXISTS bar (
  a TEXT NOT NULL,
  b TEXT NOT NULL,

  FOREIGN KEY (a) REFERENCES foo (a),
  PRIMARY KEY (a, b)
);
```

The error I get is:
```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', /home/callym/.cargo/registry/src/github.com-1ecc6299db9ec823/sea-orm-codegen-0.2.3/src/entity/transformer.rs:69:83
```

Running v0.2.3

The example is maybe a bit unclear, my use case is trying to store plant information (family, genus, species, etc), where (species name) isn't unique (there's tons of plants with a species name of e.g. 'alba'), but (genus name + species name) **is** unique.

**Comment by tyt2y3** [comment]
Hi there, it certainly is valid schema. 
It is a bug within codegen. I suspect that if you move the primary key line before the foreign key line, it'd work indeed.

**Comment by billy1624** [comment]
This bug exists in MySQL? Given the MySQL style of sql statements.

**Comment by tyt2y3** [comment]
What was the story here @billy1624 
I see that it failed at unwrapping, which I think it means that we can avoid the panic by checking not none.
Is there another problem implied, given the title of the PR?

**Comment by billy1624** [comment]
It is not related to "primary & foreign key referencing same column cause panics". The bug is that I forgot to ensure the conjunct relation should compose of 2 `columns` & `ref_columns`. Fixed on commit 9e584d0

**Comment by tyt2y3** [comment]
The crash has been fixed.

You can try installing the unreleased version of `sea-orm-cli 0.2.4` by

```
cargo install sea-orm-cli --git https://github.com/SeaQL/sea-orm
```

**Comment by tyt2y3** [comment]
Released in `sea-orm-cli 0.2.4` on crates.io
